frontend/src/components/UI/Modal/Modal.jsx

```diff
@@ -1,7 +1,19 @@
 import { useEffect, useRef } from "react";
 import PropTypes from "prop-types";
-import Notification from "../Notification";
+import LogItem from "../LogItem/LogItem";
```

Comment:
@johndeange `Notification` is now the generic `LogItem` component.

Reply:
Not a hard stop, but `LogItem` kinda feels like the wrong name 🤔

Reply:
We started with `ListItem` but felt too generic and landed on `LogItem` after talking it over with @kimschulke .
😄 Suggestions are welcome.

Reply:
This is simply for the individual items (notifications) right? I also see it is directly embedded in `Modal` - should every implementation of `Modal` have a `LogItem`? Should this be a more specific `Modal` like `NotificationModal` or whatever naming scheme we fall back to? 

Might need a quick walkthrough on this change, and its use-case..

Reply:
Resolved offline